tests/ZiraLink.UnitTests/Tools/TestTools.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Domain;
+using ZiraLink.Domain.Enums;
+namespace ZiraLink.UnitTests.Tools
+{
+    public class TestTools
+    {
+
+        public static DbContextOptions<AppDbContext>? _contextOptions;
+        public static AppDbContext? _dbContext;
+
+        /// <summary>
+        /// Initialization
+        /// </summary>
+        public static void Initialize()
+        {
+            var dbContextOptionsBuilder = new DbContextOptionsBuilder<AppDbContext>();
+            dbContextOptionsBuilder.UseInMemoryDatabase("AppDbContext");
+            _contextOptions = dbContextOptionsBuilder.Options;
+            _dbContext = new AppDbContext(_contextOptions);
+            SeedData();
+        }
+
+        /// <summary>
+        /// Initializing new data
+        /// </summary>
+        public static void SeedData()
+        {
+            if (_dbContext is not null)
+            {
+                // Add new customer
+                var customer = new Customer
+                {
+                    Id = 1,
+                    ViewId = Guid.NewGuid(),
+                    Username = "TestUser",
+                    Email = "TestUser@ZiraLink.com",
+                    Name = "Test",
+                    Family = "User",
+                    ExternalId = "1"
+                };
+                _dbContext.Customers.Add(customer);
+                _dbContext.SaveChanges();
```

Comment:
SaveChanges() can be removed since another SaveChanges() is called bottom

Reply:
Done.